Batteries/Data/List/Basic.lean

```diff
@@ -3,7 +3,6 @@ Copyright (c) 2016 Microsoft Corporation. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Leonardo de Moura
 -/
-import Batteries.Data.List.Init.Lemmas
```

Comment:
This one could stay. The import is currently empty but its purpose is to be included here (and ideally nowhere else).

Reply:
Done.